Dockerfile.alpine

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2017-present Sonatype, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM docker-all.repo.sonatype.com/alpine:latest
+
+# Build parameters
+ARG IQ_SERVER_VERSION=1.159.0-01
+ARG IQ_SERVER_SHA256=9646d59bfe16c6cdf99e181618253f0dc2b09713792b419d6342b7c4f6c5fc20
+ARG TEMP="/tmp/work"
+ARG IQ_HOME="/opt/sonatype/nexus-iq-server"
+ARG SONATYPE_WORK="/sonatype-work"
+ARG CONFIG_HOME="/etc/nexus-iq-server"
+ARG LOGS_HOME="/var/log/nexus-iq-server"
+ARG GID=1000
+ARG UID=1000
+ARG TIMEOUT=600
+
+LABEL name="Nexus IQ Server image" \
+  maintainer="Sonatype <support@sonatype.com>" \
+  vendor=Sonatype \
+  version="${IQ_SERVER_VERSION}" \
+  release="1.159.0" \
+  url="https://www.sonatype.com" \
+  summary="The Nexus IQ Server" \
+  description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality" \
+  com.sonatype.license="Apache License, Version 2.0" \
+  com.sonatype.name="Nexus IQ Server image" \
+  run="docker run -d -p 8070:8070 -p 8071:8071 IMAGE" \
+  io.k8s.description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality" \
+  io.k8s.display-name="Nexus IQ Server" \
+  io.openshift.expose-services="8071:8071" \
+  io.openshift.tags="Sonatype,Nexus,IQ Server"
+
+USER root
+
+RUN apk update \
+  && apk add --no-cache curl \
+  && apk add --no-cache openjdk8-jre \
+  && apk add --no-cache git
```

Comment:
Two things here need clarification:

1. The ubi based image installs many more packages on top of the `jdk` and `git` i.e. procps gzip unzip tar shadow-utils findutils util-linux less rsync. Are we sure they are not needed?
2. All the packages we decide to add can be installed in one shot/single `apk add` command e.g. 
`apk add --no-cache curl openjdk8-jre git`

Reply:
1. I do not have an answer to this. I did test scans, support zip, upload file and everything is working.. SCM, Pr commenting.. 
2. 9654f4e0af8cb19ec94393966f1cc459100da4a4